src/main/java/com/labsynch/labseer/chemclasses/indigo/ChemStructureServiceIndigoImpl.java

```diff
@@ -276,7 +277,7 @@ public int[] searchMolStructures(String molfile, StructureType structureType, Se
 
 			return hitList;
 
-		} catch (JpaSystemException sqlException) {
+		} catch (JpaSystemException | PersistenceException sqlException) {
```

Comment:
Error we hit in testing was a PersistenceException so I added this extra catch here so that the Bingo error would get propagated up.